.github/workflows/main.yml

```
@@ -17,8 +17,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12-dev"]
-        architecture: ["x64", "x86"]
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12', '3.13-dev']
+        architecture: ['x64', 'x86']
```

```suggestion
        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12", "3.13-dev"]
        architecture: ["x64", "x86"]
```
Just keeping quote style consistent